pypeit/specobjs.py

```diff
@@ -331,6 +331,29 @@ def purge_neg(self):
             self.remove_sobj(index)
 
 
+    def make_neg_pos(self):
+        """
+        Purge negative objects from specobjs for IR reductions
+
+        """
+        # Assign the sign and the objids
+        if self.nobj > 0:
+            if self[0].PYPELINE == 'Echelle':
+                index = self.ECH_OBJID < 0
+            elif self[0].PYPELINE == 'MultiSlit':
+                index = self.OBJID < 0
+            else:
```

Comment:
elif self[0].PYPELINE == 'IFU':
                index = self.OBJID < 0

Reply:
Added, although we don't yet have negative objects for IFU data.